tests/shared/src/test/scala/scala/meta/tests/parsers/BasePositionSuite.scala

```diff
@@ -0,0 +1,65 @@
+package scala.meta.tests.parsers
+
+import munit.FunSuite
+import scala.meta.parsers.Parse
+import scala.meta.internal.inputs._
+import munit.TestOptions
+import scala.meta.internal.trees.Origin
+import munit.Location
+import scala.meta.tests.parsers.MoreHelpers
+
+abstract class BasePositionSuite extends ParseSuite {
+  import scala.meta._
+  def defaultDialect: Dialect = dialects.Scala213
+
+  def check[T <: Tree: Parse](code: TestOptions)(implicit loc: Location): Unit = {
+    check[T](code, "")
+  }
+  def check[T <: Tree: Parse](code: TestOptions, expected: String)(implicit loc: Location): Unit = {
```

Comment:
```suggestion
  def checkPositions[T <: Tree: Parse](code: TestOptions, expected: String)(implicit loc: Location): Unit = {
```
?

Reply:
Also, could you a comment about the context of what we are testing here? so that in the future no one will have problem what is going on.

Reply:
Good idea. Done, I added a docstring explaining the motivation for these kinds of tests.